src/auto_archiver/archivers/youtubedl_archiver.py

```diff
@@ -104,22 +211,11 @@ def download(self, item: Metadata) -> Metadata:
             except Exception as e:
                 logger.error(f"Error processing entry {entry}: {e}")
 
-        # extract comments if enabled
-        if self.comments:
-            result.set("comments", [{
-                "text": c["text"],
-                "author": c["author"], 
-                "timestamp": datetime.datetime.fromtimestamp(c.get("timestamp"), tz = datetime.timezone.utc)
-            } for c in info.get("comments", [])])
-
-        if (timestamp := info.get("timestamp")):
-            #TODO: fix deprecated timestamp, 
-            timestamp = datetime.datetime.fromtimestamp(timestamp, tz = datetime.timezone.utc).isoformat()
-            result.set_timestamp(timestamp)
-        if (upload_date := info.get("upload_date")):
-            upload_date = datetime.datetime.strptime(upload_date, '%Y%m%d').replace(tzinfo=datetime.timezone.utc)
-            result.set("upload_date", upload_date)
+        result = self.add_metadata(ie, info, url, result)
+        extractor_name = "yt-dlp"
+        if ie:
+            extractor_name += f"--{ie}IE"
```

Comment:
as this is mostly user-facing we should shorten/simplify as much as possible, 
```suggestion
            extractor_name += f"_{ie}"
```